src/handlers/assets.js

```diff
@@ -21,6 +22,15 @@ const nativeAssetsPerNetwork = {
   [networkTypes.ropsten]: ETH_ADDRESS,
 };
 
+export function isL2Asset({ type }) {
```

Comment:
Can we have this like the switch case in `isL2Network` in `web3.js`; also can we just pass in the `type` directly instead of the entire asset object